src/chains/arbitrum/index.ts

```diff
@@ -1,8 +1,10 @@
 import { arbitrum as arbitrumMetadata } from '@wagmi/chains';
 import { Chain } from '@/chains';
+import { opcodes } from './vm/opcodes';
```

Comment:
_(arbitrary comment location to start a thread)_

Merges to main are auto-deployed, so based on the open PRs my impression is that we should:
- Finish this one and get it merged, followed by #13. Both of these should have no UI impact.
- Merged #14, but we'll add a `{false && ...` type of thing to hide the new UI components for now
- Another PR (or a few depending on how we want to split it up) to finish the opcodes then unhide the UI components

Let me know if that aligns with what you had in mind

Reply:
That's exactly what I had in mind. I'll address your comments tomorrow morning (my timezone is paris - utc+2). Thanks :)

Reply:
Regarding the last PR that will add the config for all the remaining opcodes, do you want me to break it down into smallr PRs (maybe by instruction group as done in [ethereum/execution-specs](https://github.com/ethereum/execution-specs/tree/master/src/ethereum/shanghai/vm/instructions)?)  or one huge PR is fine? I know there are lots of opcodes to add and the PR can become huge and hard to review...

Reply:
Whatever is easiest for you! If you'd rather knock it all out with one big PR because that's easier I'm ok with that

Reply:
Will work on this once #18 is merged :)